src/core/public/doc_links/doc_links_service.ts

```diff
@@ -18,16 +18,19 @@
  */
 
 import { deepFreeze } from '@kbn/std';
-import { InjectedMetadataSetup } from '../injected_metadata';
+import { InjectedMetadataSetup, InjectedMetadataStart } from '../injected_metadata';
 
-interface StartDeps {
+interface SetupDeps {
   injectedMetadata: InjectedMetadataSetup;
 }
 
+interface StartDeps {
+  injectedMetadata: InjectedMetadataStart;
+}
+
 /** @internal */
 export class DocLinksService {
-  public setup() {}
-  public start({ injectedMetadata }: StartDeps): DocLinksStart {
+  public setup({ injectedMetadata }: SetupDeps): DocLinksSetup {
```

Comment:
It didn't require it, but I wanted to register this toast logic in `setup`, which required I expose these values during `setup`. I think I'm going to put this back in `start` since it requires a lot of unnecessary changes.